content/en/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -9,22 +9,24 @@ weight: 90
 ---
 
 {{% capture overview %}}
-A service account provides an identity for processes that run in a Pod.
+A _service account_ provides an identity for processes that run in a Pod.
 
-*This is a user introduction to Service Accounts. See also the
-[Cluster Admin Guide to Service Accounts](/docs/reference/access-authn-authz/service-accounts-admin/).*
+*This is a user introduction to Service Accounts. See also [Managing Service Accounts](/docs/reference/access-authn-authz/service-accounts-admin/), aimed at cluster operators.*
+
+When you, a human, access the cluster&mdash;for example, using `kubectl`)&mdash;you are
```

Comment:
- there is a bracket mismatch;
- I don't think we prefer any dashes in the text.